### DISCUSSION TITLE
How do i create a policies.json file

### DISCUSSION THREAD

**Discussion by taghack** [jlesage/docker-firefox#251] [Q&A]
Hi, I am trying to lock down the Firefox instance inside the docker container. This can easily be done with a policies.json file. Typically, it should reside within /etc/firefox/policies/policies.json The question is, how do I access that path in a docker container to create the file? Thank you!

**Comment by jlesage** [comment]
I would create the file on your machine and then map it to the container during its creation (the `-v` parameter of the `docker run` command). If you create it inside the container directly, it will be lost every time you re-create the container (to update the image for example).
